web/styles/compose.css

```diff
@@ -55,8 +55,12 @@
             border-radius: 3px;
         }
 
-        #left_bar_compose_reply_button_big {
+        .compose_reply_button_wrapper {
```

Comment:
I'm all for losing the ID here, but let's use Zulip-style hyphenated selectors (`.compose-reply-button-wrapper`).

Reply:
Renamed to `.compose-reply-button-wrapper`. Due to some reason, many of the class names over at `compose.hbs` use the snake-case convention, which led me to spontaneously using that in the beginning. Maybe since we use them mostly for the logic than styling? Or maybe we should consider refactoring them to reinforce the Zulip-styling?